src/lib/executors/Node.ts

```diff
@@ -631,6 +644,10 @@ export default class Node extends Executor<NodeEvents, Config, NodePlugins> {
         }
       }
 
+      this.config.node.tsconfig
```

Comment:
I think an `if/else` may be clearer for imperative code. Also, we should provide some way to disable ts-node/register, like by setting `config.node.tsconfig` to `false` or something, in case someone doesn't want it loaded.

Reply:
👍